stock_move_backdating/wizard/stock_partial_picking_view.xml

```diff
@@ -1,17 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <openerp>
     <data>
-
-        <record id="stock_partial_picking_form" model="ir.ui.view">
-            <field name="name">stock.partial.picking.form</field>
-            <field name="model">stock.partial.picking</field>
-            <field name="inherit_id" ref="stock.stock_partial_picking_form" />
-            <field name="arch" type="xml">
-                <xpath expr="//field[@name='move_ids']/tree//field[@name='product_uom']" position="after">
-                    <field name="date_backdating" on_change="on_change_date_backdating(date_backdating)"/>
-                </xpath>
-            </field>
-        </record>
-
+      <record id="stock_partial_picking_form" model="ir.ui.view">
```

Comment:
You changed the indentation to 2 it was 4 but some elements are still indented to 4 `<openerp><data>`.